pages/app/models/refinery/page.rb

```diff
@@ -8,35 +8,27 @@ module Refinery
   class Page < Core::BaseModel
     extend FriendlyId
 
-    translates :title, :menu_title, :custom_slug, :slug, :include => :seo_meta
+    extend Mobility
+    translates :title, :menu_title, :custom_slug, :slug, :browser_title, :meta_description
 
-    attribute :title
-    attribute :menu_title
-    attribute :custom_slug
-    attribute :slug
-
-    after_save { translations.collect(&:save) }
+    default_scope { i18n }
```

Comment:
This might fix some issues, but ideally it would be better not to use it if possible.